### ISSUE TITLE
Prepare Next Submission using Latest Dataset

### ISSUE THREAD

**Issue by alizat** [Dainanahan/IDG-DREAM-Drug-Kinase-Binding-Prediction-Challenge#14]
Using the training and test datasets generated by **dataset2.R**, a new file called **submission3.R** is to be created and it will do the following:
- read training and test sets
- train on training set and predict on test set
- save test set prediction in a submission file called **round_1_submission_3.csv**

The files, **submission1.R** and **submission2.R**, may be used as guidance while writing **submission3.R**. Note that, prior to working on this issue, the **train.csv** file needs to be generated first (by running lines 742-762 of **dataset2.R**).
